python/ray/tune/experimental/output.py

```diff
@@ -390,18 +396,21 @@ def _get_trial_table_data(
     trials_by_state = _get_trials_by_state(trials)
 
     # get the right metric to show.
-    metric_keys = [
-        k
-        for k in metric_keys
+    if not isinstance(metric_keys, dict):
+        metric_keys = {k: k for k in metric_keys}
+    metric_keys = {
+        k: v
+        for k, v in metric_keys.items()
         if any(
             unflattened_lookup(k, t.last_result, default=None) is not None
             for t in trials
         )
-    ]
+    }
 
     # get header from metric keys
     formatted_metric_columns = [
-        _max_len(k, max_len=max_column_length, wrap=wrap_headers) for k in metric_keys
+        _max_len(v, max_len=max_column_length, wrap=wrap_headers)
+        for v in metric_keys.values()
     ]
```

Comment:
https://github.com/ray-project/ray/pull/45122/files#diff-4623db4f66d85c361967ca9abf09f1e2c37f956af83ea12ede3a622c01b47de0R420-R423

I think the lines highlighted no longer need the `DEFAULT_COLUMNS` conditional branch. Can just use the values in `formatted_metric_columns`.

Reply:
I'm not sure I understand. What about the mappings to to-be-dispayed names that `DEFAULT_COLUMNS` currently provides? Can we please do this in a separate PR? This has nothing to do with what this PR intends to do.